charts/stable/lidarr/templates/common.yaml

```diff
@@ -1 +1,36 @@
+{{/* Make sure all variables are set properly */}}
+{{- include "common.values.setup" . }}
+
+{{/* Append the hardcoded settings */}}
+{{- define "lidarr.harcodedValues" -}}
+{{ if .Values.exporter.enabled }}
+additionalContainers:
+  exporter:
+    name: exporter
+    image: "{{ .Values.exporter.image.repository }}:{{ .Values.exporter.image.tag }}"
+    imagePullPolicy: {{ .Values.exporter.image.pullPolicy }}
+    args: ["exportarr", "lidarr"]
+    env:
+      - name: URL
+        value: "http://localhost"
+      - name: CONFIG
+        value: "/config/config.xml"
+      - name: PORT
+        value: "{{ .Values.exporter.env.port }}"
+      - name: ENABLE_ADDITIONAL_METRICS
+        value: "{{ .Values.exporter.env.additionalMetrics }}"
+      - name: ENABLE_UNKNOWN_QUEUE_ITEMS
+        value: "{{ .Values.exporter.env.unknownQueueItems }}"
+    ports:
+      - name: metrics
+        containerPort: {{ .Values.exporter.env.port }}
+    volumeMounts:
+      - name: config
+        mountPath: /config
+        readOnly: true
```

Comment:
This can potentially break when there is the `config` persistence item is disabled.

Reply:
ok should be fixed